src/main/resources/view/StatsWindow.fxml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.Scene?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.stage.Stage?>
+
+<?import javafx.scene.*?>
```

Comment:
* import statement

observation:
seems like checkstyle does check this in fxml files

Reply:
not an issue at the moment.
question: are we allowed to modify checkstyle.xml

no need to resolve now